consensus/core/src/metrics.rs

```diff
@@ -37,161 +59,189 @@ pub(crate) fn test_metrics() -> Arc<Metrics> {
+                "The time taken between block creation and block commit.",
                 registry,
             )
-            .unwrap(),
-            core_lock_dequeued: register_int_counter_with_registry!(
-                "core_lock_dequeued",
-                "Number of dequeued core requests",
+                .unwrap(),
+            block_proposed_total: register_int_counter_vec_with_registry!(
+                "block_proposed",
+                "Total number of block proposals. If force is true then this block has been created forcefully via a leader timeout event.",
+                &["force"],
                 registry,
             )
-            .unwrap(),
-            leader_timeout_total: register_int_counter_with_registry!(
-                "leader_timeout_total",
-                "Total number of leader timeouts",
-                registry,
-            )
-            .unwrap(),
-            threshold_clock_round: register_int_gauge_with_registry!(
-                "threshold_clock_round",
-                "The current threshold clock round. We only advance to a new round when a quorum of parents have been synced.",
-                registry,
-            ).unwrap(),
-            suspended_blocks: register_int_counter_vec_with_registry!(
-                "suspended_blocks",
-                "The number of suspended blocks. The counter is reported uniquely, so if a block is sent for reprocessing while alreadly suspended then is not double counted",
-                &["authority"],
-                registry,
-            ).unwrap(),
-            unsuspended_blocks: register_int_counter_vec_with_registry!(
-                "unsuspended_blocks",
-                "The number of unsuspended blocks",
-                &["authority"],
-                registry,
+                .unwrap(),
+            block_size: register_histogram_with_registry!(
+                "block_size",
+                "The size (in bytes) of proposed blocks",
+                SIZE_BUCKETS.to_vec(),
+                registry
             ).unwrap(),
-            // TODO: add a short status label.
-            invalid_blocks: register_int_counter_vec_with_registry!(
-                "invalid_blocks",
-                "Number of invalid blocks per peer authority",
-                &["authority", "source"],
-                registry,
-            )
-            .unwrap(),
             block_timestamp_drift_wait_ms: register_int_counter_vec_with_registry!(
                 "block_timestamp_drift_wait_ms",
                 "Total time in ms spent waiting, when a received block has timestamp in future.",
                 &["authority"],
                 registry,
             )
             .unwrap(),
+            blocks_per_commit_count: register_histogram_with_registry!(
+                "blocks_per_commit_count",
+                "The number of blocks per commit.",
+                registry,
+            )
+                .unwrap(),
```

Comment:
super nit: indentation is a bit more inconsistent in this file now.

Reply:
Thanks for pointing out, I'll fix that manually as cargo fmt doesn't take care of that